lib/classes/ConfigSchemaHandler/resolveAjvValidate.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const Ajv = require('ajv').default;
+const objectHash = require('object-hash');
+const path = require('path');
+const os = require('os');
+const standaloneCode = require('ajv/dist/standalone').default;
+const fs = require('fs');
+const requireFromString = require('require-from-string');
+const fse = require('fs-extra');
+const deepSortObjectByKey = require('../../utils/deepSortObjectByKey');
+const ensureExists = require('../../utils/ensureExists');
+
+const getValidate = async (schema) => {
+  const cacheDir = path.resolve(os.homedir(), '.serverless/schemas');
```

Comment:
I would stick to `.serverless/artifacts` folder, and place it per AJV version (e.g. AJV may be upgraded with a  bug fix, and in such case we should allow to generate new `validate` method and not stick to old one). So cache dir is as:

```javascript
cacheDir = path.resolve(os.homedir(), `.serverless/artifacts/ajv-validate-${require('ajv/package').version}`);
```

Reply:
changed :+1: